payload.go

```diff
@@ -137,7 +144,15 @@ func ProcessPayload(e []github.Event, u User) ([]ActivityPayload, error) {
 			if comment, ok := rawPayload["comment"].(map[string]interface{}); ok {
 				payload.Body = comment["body"].(string)
 			}
-			activityPayloads = append(activityPayloads, payload)
+
+			// Save the paylod to the db events table
+			err := payload.Create()
+			if err != nil {
+				//Assuming fail due to the unique constraint
+			} else {
+				activityPayloads = append(activityPayloads, payload)
+			}
+
```

Comment:
Yea, sorry. I forgot I was going to just loop through the activityPayloads before we return it and do this. Any way to easily pop the ones that fail so I don't have to write to a new slice?